node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
> Why should the end developer be concerned with the "value"? They shouldn't need to care—the LED can only only be "on" or "off"

That is my strongest reason for adding a convenience method for checking the `value` of the LED. None of the existing public methods reference a 'value' for the LED, so documenting that property does not seem consistent with how they control and interact with LED at the moment. 

If the end developer could check the state of the LED by using something like `tessel.led[0].isOn()` and get a 'true' or 'false' response from that, then it fits in with the goal for them to use `on()`, `off()`, and `toggle()`, as opposed to considering how the value of 0 or 1 reflects the state of the LED in their program. It might also be confusing that changing that property manually does not change the physical state of the LED.